lib/puppet/property.rb

```diff
@@ -424,6 +424,7 @@ def set(value)
     name = self.class.value_name(value)
     if method = self.class.value_option(name, :method) and self.respond_to?(method)
       begin
+        puts "sending #{method.inspect}"
```

Comment:
I very carefully `git add -p`'d the bits I wanted, then used my "AMEND IT ALL INTO ONE COMMIT AND FORCE PUSH" script 


Because I am smart